src/NSwag.AspNetCore/NSwag.AspNetCore.nuspec

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0"?>
+<package>
+    <metadata>
+        <id>$id$</id>
+        <version>$version$</version>
+        <authors>$authors$</authors>
+        <description>$description$</description>
+        <tags>Swagger Documentation WebApi AspNet TypeScript CodeGen</tags>
+        <projectUrl>https://github.com/NSwag/NSwag</projectUrl>
+        <licenseUrl>https://github.com/NSwag/NSwag/blob/master/LICENSE.md</licenseUrl>
```

Comment:
@RicoSuter does it matter (much) that I'm using old GitHub locations for this repo i.e. should `projectUrl` be https://github.com/RicoSuter/NSwag ?

Reply:
We should use the correct URL, why using the old one?

Reply:
This matches what I see in your released packages. I'll update.